src/content/docs/ru/tutorials/add-view-transitions.mdx

```diff
@@ -0,0 +1,446 @@
+---
+title: Учебное пособие - Расширение с помощью анимации переходов
+description: >-
+  Добавляем анимацию переходов в код учебного пособия по созданию блога.
+i18nReady: true
+---
+import { Steps } from '@astrojs/starlight/components';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro';
+import Spoiler from '~/components/Spoiler.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+import ReadMore from '~/components/ReadMore.astro'
+
+
+**Анимации переходов** — это способ управлять тем, что происходит, когда посетители переходят между страницами вашего сайта. API View Transitions в Astro позволяет добавлять дополнительные возможности навигации, включая плавные переходы и анимацию страниц, управление стопкой посещённых страниц в истории браузера и предотвращение обновления всей страницы, чтобы сохранить некоторые элементы и состояние страницы при обновлении отображаемого контента.
+
+<PreCheck>
```

Comment:
Because the PreCheck block starts with word "Приготовьтесь..", we have to change wording a bit
```suggestion
  - Импортировать и добавить маршрутизатор `<ViewTransitions />` в общий элемент `head`
  - Добавить прослушиватели событий во время процесса навигации для запуска `<script>` при необходимости
  - Добавить анимации перехода между страницами с использованием директив перехода
  - Исключить из клиентской маршрутизации ссылку на отдельную страницу
<PreCheck>
```